### DISCUSSION TITLE
Not able to use URH with RSPdx

### DISCUSSION THREAD

**Discussion by jpfrance** [jopohl/urh#1006] [SDR]
First, let me advocate I'm a newbie in URH usage. I've got an RSPdx that I'm using with other aplications and I wanted to use URH for its reverse engineering capabilities . I've downloaded URH from https://github.com/jopohl/urh/releases (64bits version for windows) and  API/HW Driver 3.11 | RSP Control Library + Drive from https://www.sdrplay.com/downloads/. 

According to  URH Options, SDRplay is supported . But when I launch URH Spectrum analyser , I got the following: " SDRPlay-SET DEVICE NUMBER to 0: HARDWARE ERROR (7).  

May be the solution si very simple, but I recognize that I'm a bit lost. Is there somebody around who could advise. 

Many thanks

Windosws 11

URH for Windows 64bits/32 bits version (I've tried both)

**Comment by farfocel69** [comment]
Hi, I'm having same problem. Have you found any solution?

**Reply by stevenjoezhang** [reply]
Testing of this PR is welcome: https://github.com/jopohl/urh/pull/1188
